portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/public/js/add.js

```diff
@@ -0,0 +1,119 @@
+$(document).ready(function () {
+    $('#domainSelector option[value=' + $('#domainSelector').attr('data-primary') + ']').prop('selected', 'selected');
+    var primaryDomain = $('#domainSelector').val();
+    $("#domain").val(primaryDomain);
+    //load domain drop down when page loads
+    displayVals();
+    $('#domainSelector').change(function () {
+        var domain = $(this).val();
+        $("#domain").val(domain);
+    });
+    $('#verificationSelector').change(displayVals);
+});
+$(window).load(function () {
+    jQuery.validator.addMethod("noSpace", function (value, element) {
+        return value.indexOf(" ") < 0 ;
+    }, "Please don't enter spaces");
+});
+function displayVals() {
+    if ($("#verificationSelector option:selected").text() === "With password") {
```

Comment:
Checking the text value is not suitable. We should give flexibility to change them with i18n as well. Use some other method to check the condition.